requirements.txt

```
@@ -1,7 +1,7 @@
 black==22.3.0
 flake8==5.0.4
 flake8-tidy-imports==4.6.0
-graviton@git+https://github.com/algorand/graviton@v0.5.0
+graviton@git+https://github.com/algorand/graviton@f6e9dfa965afd9588bfaf7325afe5f8c8edafae3
```

```suggestion
graviton@git+https://github.com/algorand/graviton@v0.6.0
```
Change after [graviton #36](https://github.com/algorand/graviton/pull/36/files) merged and the new tag is added